### DISCUSSION TITLE
Map Element - Multiple Tile URLs/Attributions

### DISCUSSION THREAD

**Discussion by mmalessa80** [Budibase/budibase#6317] [Help] [answered]
Having worked with Leaflet JS and OpenStreetMaps tile layer before, I had created a map that had the OSM tile layer as the 'Street Level' option and imported an ESRI satellite tile layer as the 'Satellite' option for basemaps.

It appears with the standard map element it is only possible to include a single base map tile layer vs multiple tile layers and the ability to toggle between them.

Could someone tell me if that is in fact the case or how can I incorporate multiple basemap options (i.e. multiple tile URLs each with their attribution)?

![image](https://user-images.githubusercontent.com/106758851/173641053-2d6645e6-7109-40ae-9228-5eed06df0606.png)

![image](https://user-images.githubusercontent.com/106758851/173641135-4a3e9947-7fd8-4214-aa2e-c83cdf0514d3.png)

**Comment by melohagan** [comment, marked as answer]
Hi @mmalessa80 

I think you are correct, that multiple layered tilesets cannot be used at the moment - feel free to raise a feature request!
